oops_program_1.cpp

```diff
@@ -0,0 +1,58 @@
+#include<iostream>
+Class student
+{
+   Protected:
+       int roll_number;
+   public:
+       void get_number(int);
+       void put_number(void);
+ }
+Void student::get_number(int a)
+{
+    Roll_number=a;
```

Comment:
c++ is case sensitive right? class attribute declared as int roll_number and here is being accessed as Roll_number. 
Did you compile this code? Did it work?